flowauth/frontend/src/UserAdminDetails.js

```
@@ -131,6 +138,11 @@ class UserAdminDetails extends React.Component {
         })
         .then(json => {
           onClick();
+        })
+        .catch(err => {
+          if (err.code === 400) {
```

Need to rethrow the error if it _isn't_ a 400